cmd/datamon/cmd/config.go

```diff
@@ -15,14 +15,13 @@ import (
 type CLIConfig struct {
 	// bug in viper? Need to keep names of fields the same as the serialized names..
 	Credential string `json:"credential" yaml:"credential"` // Credentials to use for GCS
-	Config     string `json:"config" yaml:"config"`         // Config for datamon
+	Config     string `json:"config" yaml:"config"`         // Config bucket for datamon
```

Comment:
still makes little sense to me to call the bucket where contexts are stored the "config bucket" .. would think "context bucket" easier to remember and can only suppose that we're allowing for the possibility store config info other than context in this bucket.

Reply:
yeah I just wanted to strike the difference with the "local config file". If you feel strong about some clever formulation for it, feel free to change it.